docs/content/en/docs/Components/thanos.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Integrate Thanos with Kurator"
+linkTitle: "Integrate Thanos with Kurator"
+description: >
+    Integrate Thanos with Kurator?
+---
+
+### What is Thanos
+
+[Thanos](https://thanos.io) provides a global query view, high availability, data backup with historical, cheap data access as its core features in a single binary.
+
+In this task, we will show you how to install Thanos with kurator.
+
+### Prerequisites
+
+This task requires you have installed karmada and have joined at least one member cluster.
```

Comment:
should be updated after https://github.com/kurator-dev/kurator/pull/78#issuecomment-1253282709 merged